### ISSUE TITLE
text and icons, some BE fields disappeared after update

### ISSUE THREAD

**Issue by gilas1959** [benjaminkott/bootstrap_package#604]
# Bug Report
PHP version 7.2
TYPO3 Versione 9.5.x
bootstrap package: 10.1.x (latest)

## Prerequisites
Problem related to TYPO3 v9.5 LTS

## Description
some BE fields, running the TEXT with ICON CType, after upgrading a site from TYPO3 8.7 to TYPO3 9.5 are not available. See enclosed screenshot.

![noiconset](https://user-images.githubusercontent.com/6662802/49215088-474c6d80-f3c8-11e8-86ac-64ea6b065b08.gif)

**Comment by benjaminkott** [comment]
Looks strange, there are a lot of fields missing in this view. 
Did you run the migration wizards?
Can you please provide a sample row from the database?

**Comment by gilas1959** [comment]
yes, I ran the update wizard and db update, etc. but nothing changes... what do you mean for "sample row from the database"?  thank you

**Comment by gilas1959** [comment]
moreover there is some things that are not clear about the post-update. If I want to use LESS + bootstrap 3, I suppose that library "Bootsrap Package: Bootstrap 3.x (bootstrap package)". This was done correctly but there is no way to see any icon rendered. It seems that the font with glyphicons it is not availbale/loaded. As I use sysnews with twitter style, my icons are disappeared from the frontend....

**Comment by benjaminkott** [comment]
Would like to have a look what's stored in the database.

SELECT uid, pid, icon, icon_position, icon_size, icon_type, icon_color, icon_background, icon_set, icon_file FROM `tt_content` WHERE `CType` LIKE 'texticon'

![image](https://user-images.githubusercontent.com/3243119/49397259-52dac400-f73b-11e8-9ce9-ee5ea4243a4a.png)

**Comment by gilas1959** [comment]
very good. Today I have installed another site that uses same TYPOSCRIPT, page variables and so on. This time the new site works correctly. In the other site I suspect that the lack of the fields is due to some installation of bootstrap 10 and revert to v. 9.x and new to 10.  So I will uninstall and start again from ZERO. 
I wil inform you as soon as possible., thank you very much for your "Glyphicon usage" above!

**Comment by benjaminkott** [comment]
Please ping me on slack, I think we only need to reactivate the upgrade wizards. We can have a look together.

**Comment by benjaminkott** [comment]
Sadly, not that I am aware of since it was not designed to work that way. I´ve built some stuff that could be used to achieve similar - but you need a template. But it can live somewhere and you do not need to override templates for this. I´ve called it blocks and is intended for standalone usage.

Please see:
https://github.com/benjaminkott/bootstrap_package/blob/master/Configuration/TypoScript/Helper/Block.typoscript

**What you could do!**
constants.typoscript
```
plugin.bootstrap_package_blocks.view.templateRootPath = fileadmin/blocks/
```

setup.typoscript
```
lib.glyphicon =< lib.block
lib.glyphicon {
    templateName = Glyphicon
    dataProcessing {
        10 = BK2K\BootstrapPackage\DataProcessing\StaticFilesProcessor
        10.files.0 = EXT:bootstrap_package/Resources/Public/Images/Icons/Glyphicons/adjust.svg
    }
}
```

fileadmin/blocks/Glyphicon.html
```
<html xmlns:f="http://typo3.org/ns/TYPO3/CMS/Fluid/ViewHelpers" data-namespace-typo3-fluid="true">
<f:if condition="{icon.0.extension} === svg">
    <f:then>
        <bk2k:inlineSvg image="{icon.0}" height="64" width="64"/>
    </f:then>
    <f:else>
        <f:image image="{icon.0}" height="64" width="64"/>
    </f:else>
</f:if>
</html>
```

**Comment by gilas1959** [comment]
first check: under t3 9.5.x and bs 10.0.0 I can find only one wizard marked as DONE (related to Carousel) , but not yours:
![image](https://user-images.githubusercontent.com/6662802/49506569-a7d32300-f87e-11e8-8c9e-ef65b07244a3.png)

uprading to 10.0.3 nothing changes, the same:
![image](https://user-images.githubusercontent.com/6662802/49506633-d7822b00-f87e-11e8-800f-4b1022b206d6.png)

as you are doing some tests under v 8.7.20 I suppose that there is something that files under t3 9.5

**Comment by gilas1959** [comment]
and this is my palettes configuration

![image](https://user-images.githubusercontent.com/6662802/49506994-d0a7e800-f87f-11e8-846c-5c2a8545c710.png)

**Comment by gilas1959** [comment]
I have upgraded bs from 8.x to 9 then to 10,0.3. Reinstalled bs 9 and then 10.0.0 and newly to 10.0.3 but nothing changed.

**Comment by benjaminkott** [comment]
Hey, did you read my e-mail about the "rte_custom" extension that is causing the fields to be missing?
